### ISSUE TITLE
ERROR ... machine in headless mode

### ISSUE THREAD

**Issue by AmandaRossimandi** [ashemery/CuckooVM#8]
Good morning, I 've this problem:

_2020-07-21 13:57:12,974 [cuckoo.core.scheduler] ERROR: Error starting Virtual Machine! VM: cuckoo1, error: VBoxManage failed starting the machine in headless mode. Are you sure your machine is still functioning correctly when trying to use it manually? Error: VBoxManage: error: The VM session was aborted
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component SessionMachine, interface ISession_ 

then, probabily as a consequence this:
_2020-07-21 13:57:13,579 [cuckoo.processing.debug] ERROR: Error processing task #7: it appears that the Virtual Machine hasn't been able to contact back to the Cuckoo Host......_

Following check and screenshot of my controls:
3D/2D disabled on VBox , ping of IP 192.168.56.10 is ok reachable, changed virtualbox.conf mode from headless to gui generate error suggesting to use headless mode. Manual start of VM with vboxheadless -s win7 it's OK, Windows Firewall is disable.
 
![logCuckooError](https://user-images.githubusercontent.com/63255504/88061746-3685d800-cb68-11ea-8224-1edac4c2953a.png)
![errorOnCuckooWeb](https://user-images.githubusercontent.com/63255504/88061792-41406d00-cb68-11ea-8a2e-2b0ee991349d.png)
![headlessWork](https://user-images.githubusercontent.com/63255504/88061912-6d5bee00-cb68-11ea-9c15-affd25bf8ab7.png)

Do you have some ideas?   thanks regards

**Comment by vnc2017** [comment]
@AmandaRossimandi I had the same issue. 

Below is what I did to fix it.

Started the windows vm through virtualbox.
Once the vm is loaded. python window automatically loads since the agent.py is in startup folder.
I closed it and restarted it ( let the python window be still open ). then in new cmd prompt pinged 192.168.56.1 (cuckoo vm). from cuckoo vm pinged the 192.168.56.10 ( windows 7 vm ). Confirmed both are pinging each other. 
After the above check took a snaphot and named base3. 

Last step  sudo nano ~/.cuckoo/conf/virtualbox.conf
goto snapshot line and change base2 to base3.

New terminal n cuckoo vm - cuckoo -d and in another terminal cuckoo web.
In the web browser submit your malware sample and wait for analysis results. Enjoy!!
